sessiontxn/isolation/repeatable_read_test.go

```diff
@@ -346,6 +347,8 @@ func TestOptimizeWithPlanInPessimisticRR(t *testing.T) {
 	stmt, err := parser.New().ParseOneStmt("delete from t where id = 1", "", "")
 	require.NoError(t, err)
 	compareTs := getOracleTS(t, se)
+	err = provider.OnStmtStart(context.TODO())
```

Comment:
Is necessary to collect these sqls in a list like:

```
cases := []sturct{
  sql string
  shouldOptimize bool
} {
  {
    "sql": ...
    "shouldOptimize": ...
  }
}
```

This can be more clear and easy to add new cases

Reply:
Done